Traceback (most recent call last):
  File "train.py", line 11, in <module>
    import pandas as pd
  File "/usr/local/lib/python3.6/dist-packages/pandas/__init__.py", line 17, in <module>
    "Unable to import required dependencies:\n" + "\n".join(missing_dependencies)
ImportError: Unable to import required dependencies:
numpy: 

IMPORTANT: PLEASE READ THIS FOR ADVICE ON HOW TO SOLVE THIS ISSUE!

Importing the numpy c-extensions failed.
- Try uninstalling and reinstalling numpy.
- If you have already done that, then:
  1. Check that you expected to use Python3.6 from "/usr/bin/python3",
     and that you have no directories in your PATH or PYTHONPATH that can
     interfere with the Python and numpy version "1.18.1" you're trying to use.
  2. If (1) looks fine, you can open a new issue at
     https://github.com/numpy/numpy/issues.  Please include details on:
     - how you installed Python
     - how you installed numpy
     - your operating system
     - whether or not you have multiple versions of Python installed
     - if you built from source, your compiler versions and ideally a build log

- If you're working with a numpy git repository, try `git clean -xdf`
  (removes all files not under version control) and rebuild numpy.

Note: this error has many possible causes, so please don't comment on
an existing issue about this - open a new one instead.

Original error was: PyCapsule_Import could not import module "datetime"

Traceback (most recent call last):
  File "score.py", line 11, in <module>
    import pandas as pd
  File "/usr/local/lib/python3.6/dist-packages/pandas/__init__.py", line 11, in <module>
    __import__(dependency)
  File "/usr/local/lib/python3.6/dist-packages/numpy/__init__.py", line 142, in <module>
    from . import core
  File "/usr/local/lib/python3.6/dist-packages/numpy/core/__init__.py", line 78, in <module>
    from . import numeric
  File "/usr/local/lib/python3.6/dist-packages/numpy/core/numeric.py", line 29, in <module>
    from . import shape_base
  File "/usr/local/lib/python3.6/dist-packages/numpy/core/shape_base.py", line 14, in <module>
    from . import fromnumeric as _from_nx
  File "/usr/local/lib/python3.6/dist-packages/numpy/core/fromnumeric.py", line 838, in <module>
    def sort(a, axis=-1, kind=None, order=None):
  File "/usr/local/lib/python3.6/dist-packages/numpy/core/overrides.py", line 190, in decorator
    public_api = scope[implementation.__name__]
KeyboardInterrupt
